HW5.py

```diff
@@ -0,0 +1,586 @@
+import os
```

Comment:
I think you may have forgotten to delete this (as you wanted to change `HW5.py` to `midterm.py`, and this file's just like the later).